lightning/src/chain/keysinterface.rs

```diff
@@ -319,13 +319,13 @@ pub trait ChannelKeys : Send+Clone {
 	/// protocol.
 	fn sign_channel_announcement<T: secp256k1::Signing>(&self, msg: &UnsignedChannelAnnouncement, secp_ctx: &Secp256k1<T>) -> Result<Signature, ()>;
 
-	/// Set the remote channel basepoints and counterparty/local_to_self_delay.
+	/// Set the counterparty channel basepoints and counterparty_selected/locally_selected_contest_delay.
```

Comment:
Any reason to prefer "locally" over "holder"?

Reply:
Add a cleanup commit, iteration from a previous round of review.